### PR TITLE
PLF-8735 Install Drives application in app center by default

### PR DESCRIPTION
**Pull request by boubaker**

### CODE REVIEW

File: ecm-wcm-extension/src/main/webapp/WEB-INF/conf/wcm-extension/portal/portal/global/navigation.xml [hunk 1]
```diff
@@ -24,10 +24,10 @@
   <priority>3</priority>
   <page-nodes>
     <node>
-      <name>documents</name>
-      <label>#{portal.global.documents}</label>
+      <name>drives</name>
```
Comment (ahamdi) [inline ecm-wcm-extension/src/main/webapp/WEB-INF/conf/wcm-extension/portal/portal/global/navigation.xml]: Why did you change the navigation from documents to drives ?

Comment (boubaker) [reply ecm-wcm-extension/src/main/webapp/WEB-INF/conf/wcm-extension/portal/portal/global/navigation.xml]: As requested by PO in fact ;-)
